BossMod/Modules/Shadowbringers/Foray/CriticalEngagement/CE11ShadowOfDeathHand.cs

```diff
@@ -78,7 +79,27 @@ class BroadsideBarrage : Components.SelfTargetedAOEs
     {
         public BroadsideBarrage() : base(ActionID.MakeSpell(AID.BroadsideBarrage), new AOEShapeRect(40, 20)) { }
     }
+    class BroadsideBarrageKB : Components.Knockback
```

Comment:
What's the point of this module? The point is to avoid AOE, it doesn't matter what exactly it does on failure (kill, apply vuln, knock back into deathwall) - there are tons of AOEs that knockback into deathwall on failure, I tend not to represent that at all.

Knockback components are created for when they are actual mechanics that you need to take into account.